### PR TITLE
Checking null is bad

### PR DESCRIPTION
**Pull request by Bod9001**
removes that from staging
assuming that staging is running at 30 fps 
this should bring it up to 40 fps

### CODE REVIEW

**Comment by Bod9001** [comment]
Tested with built client , and built client connection to it, with Big explosion, with re-log and round restart

File: UnityProject/Assets/Mirror/Runtime/NetworkServer.cs [hunk 1]
```diff
@@ -1522,47 +1533,48 @@ static void BroadcastToConnection(NetworkConnectionToClient connection)
...
+	            //       TODO make this more simple / unnecessary later.#
+	            //CUSTOM UNITYSTATION CODE// commented this out \/
+	            //identity.ClearDirtyComponentsDirtyBits();
+                //}
                 // spawned list should have no null entries because we
                 // always call Remove in OnObjectDestroy everywhere.
                 // if it does have null then someone used
                 // GameObject.Destroy instead of NetworkServer.Destroy.
-                else Debug.LogWarning("Found 'null' entry in observing list for connectionId=" + connection.connectionId + ". Please call NetworkServer.Destroy to destroy networked objects. Don't use GameObject.Destroy.");
+                //else Debug.LogWarning("Found 'null' entry in observing list for connectionId=" + connection.connectionId + ". Please call NetworkServer.Destroy to destroy networked objects. Don't use GameObject.Destroy.");
```
Comment (corp-0) [inline UnityProject/Assets/Mirror/Runtime/NetworkServer.cs:1573]: Why comment this?

Comment (Bod9001) [reply UnityProject/Assets/Mirror/Runtime/NetworkServer.cs:1573]: it's the other side of the null check that I removed

Comment (corp-0) [reply UnityProject/Assets/Mirror/Runtime/NetworkServer.cs:1573]: Well yeah but checking the IsNull variable conceptually does the same and if it is true then the log is relevant, I think

Comment (Bod9001) [reply UnityProject/Assets/Mirror/Runtime/NetworkServer.cs:1573]: It's only elseon the null date Needing to be sent

**Review by corp-0 — DISMISSED** [review]

**Review by ThatDan123 — APPROVED** [review]

**Review by NoooneyDude — APPROVED** [review]

**Pull request merged**